src/parcsr_ls/par_mgr_setup.c

```diff
@@ -1536,6 +1533,19 @@ hypre_MGRSetup( void               *mgr_vdata,
                hypre_error_w_msg(0, msg);
             }
          }
+         else if (aff_solver[lev])
```

Comment:
@victorapm is this else if to bypass fsolver_mode? how does off_solver[lev] play with Relax_type[lev]?

Reply:
1. Not sure if I understood the question. but this `else if` provides the mechanism for running the F-relaxation solver created externally by the user.
2. In this code, they are not related. Do you want to create a new code for `Frelax_type` to denote "custom", i.e., "informed by the user"?